iroha/src/lib.rs

```diff
@@ -84,16 +84,20 @@ impl Iroha {
         //The id of this peer
         let iroha_peer_id = PeerId {
             address: config.torii_url.to_string(),
-            public_key,
+            public_key: [0u8; 32],
```

Comment:
Why not a public ket?

Reply:
Iroha gets trusted peers from configuration, which needs to already have public keys of all the peers. So for now the keys are all zeros. I think we need a separate PR to decouple key generation and storage from the Iroha initialization.

I suggest we leave it like that for now and create an issue.